tfs/writer.py

```diff
@@ -27,11 +27,18 @@ def write_tfs(
     colwidth: int = DEFAULT_COLUMN_WIDTH,
     headerswidth: int = DEFAULT_COLUMN_WIDTH,
     non_unique_behavior: str = "warn",
+    validate_before_writing: bool = True,
```

Comment:
why not just `validate`? it's in the writer anyway.

Reply:
See above.